Mathlib/Geometry/Manifold/Instances/Real.lean

```diff
@@ -229,6 +228,9 @@ scoped[Manifold]
     (modelWithCornersEuclideanHalfSpace n :
       ModelWithCorners ℝ (EuclideanSpace ℝ (Fin n)) (EuclideanHalfSpace n))
 
+lemma modelWithCornersEuclideanHalfSpace_zero {n : ℕ} [NeZero n] :
+    (𝓡∂ n) ⟨fun _ ↦ 0, by norm_num⟩ = 0 := rfl
```

Comment:
Why is this not
```suggestion
    (𝓡∂ n) 0 = 0 := rfl
```

Reply:
This would yield `failed to synthesize
  OfNat (EuclideanHalfSpace n) 0`

Reply:
Wait, the Euclidean half plane doesn't have a zero currently?!

Reply:
Indeed, it didn't. Added a zero instance (also for the euclidean quadrant) and simplified the proof.